chapters/equations.tex

```diff
@@ -748,7 +748,7 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 (such as simulation or linearization).
 
 The equations of a when-clause are active during initialization, if and only if they are explicitly enabled with \lstinline!initial()!, and only in one of the two forms
-\lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then! (and similarly for algorithms see below).  In this case, the when-clause equations remain active during the
+\lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then! (and similarly for \lstinline!elsewhen!  and algorithms see below).  In this case, the when-clause equations remain active during the
```

Comment:
Perhaps _when-statements_ would be more to the point, and the _see below_ probably only refers to these?
```suggestion
(where \lstinline!when! may be replaced by \lstinline!elsewhen!) \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then! (and similarly for when-statements, see below).  In this case, the when-clause equations remain active during the
```

Reply:
I first considered a text like that, but I didn't like it since there is a slight difference between elsewhen and when, in that when is always first - and I wanted to delay those obscure cases. The elsewhen-part below handles both.

Reply:
To me it just seems unfortunate that `when`/`elsewhen` causes a lack of symmetry that is only syntactical.  This is why I ended up with the formulation suggested below (_if and only if they are the first clause…_) that captures the symmetry of the semantics.